docs/product-listing.html

```diff
@@ -0,0 +1,421 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>products-Kitab</title>
+    <link rel="stylesheet" href="https://parts-builder.netlify.app/styles/main-style.css">
```

Comment:
you can also import  this link in main CSS ,, so that you can not use again and again

Reply:
Yes bro great suggestion. Will do that